### ISSUE TITLE
Data valuation methods for DenseNet121

### ISSUE THREAD

**Issue by ffeiland** [sunblaze-ucb/data-valuation#1]
Hi there,

I have a DenseNet121 for classifying images (320x320 pixels) using keras. Therefore, I have a training method that trains the model via fit_generator. 
I want to implement data valuation methods (KNN-Shap, D-Shap, TMC-Shap, LOO-values) in order to detect noisy labels. Now I have problems using your code for my model. 
What is the best way to use your code? 
e..g Can I just use my training method instead of the fit() method provided in the code?

Thanks in advance,
Regards,
Fabian
